projects/igniteui-angular/src/lib/icon/icon.component.ts

```diff
@@ -17,12 +18,12 @@ import { Subject } from 'rxjs';
  * @remarks
  *
  * The Ignite UI Icon makes it easy for developers to include material design icons directly in their markup. The icons
- * support custom colors and can be marked as active or disabled using the `isActive` property. This will change the appearance
+ * support different font-families and can be marked as active or disabled using the `active` property. This will change the appearance
```

Comment:
```suggestion
 * support different icon families and can be marked as active or disabled using the `active` property. This will change the appearance
```